WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/OldStoreStatsAndTopPerformersPeriodViewController.swift

```diff
@@ -113,7 +113,7 @@ final class OldStoreStatsAndTopPerformersPeriodViewController: UIViewController
     private let siteID: Int64
 
     /// Subscriptions that should be cancelled on `deinit`.
-    private var cancellables = [ObservationToken]()
+    private var cancellables = Set<AnyCancellable>()
```

Comment:
Nit: Should this be renamed to `subscriptions` as per our discussion a few weeks ago?

Reply:
for this class, I think I'll leave it as it is since we're deleting all the old stats classes in the near future